Changelog.md

```diff
@@ -2,15 +2,21 @@
 
 This file only reflects the changes that are made in this image. Please refer to the upstream GitLab [CHANGELOG](https://gitlab.com/gitlab-org/gitlab-foss/blob/master/CHANGELOG.md) for the list of changes in GitLab.
 
+**15.8.1**
+
+- gitlab: upgrade CE to v15.8.1
+- gitaly: upgrade to v15.8.1
+- gitlab-pages: upgrade to v15.8.1
+
 **15.8.0-1**
 
 - ruby: rollback to v2.7.7
 
-**15.8.0**
+**15.8.1**
 
-- gitlab: upgrade CE to v15.8.0
-- gitaly: upgrade to v15.8.0
-- gitlab-pages: upgrade to v15.8.0
+- gitlab: upgrade CE to v15.8.1
+- gitaly: upgrade to v15.8.1
+- gitlab-pages: upgrade to v15.8.1
```

Comment:
Please revert the changes you made, since this version numbers have to be 15.8.0!